generator/src/main/java/org/corfudb/generator/Generator.java

```diff
@@ -0,0 +1,73 @@
+package org.corfudb.generator;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import org.corfudb.generator.operations.CheckpointOperation;
+import org.corfudb.generator.operations.Operation;
+import org.corfudb.runtime.CorfuRuntime;
+
+import static java.util.concurrent.Executors.newWorkStealingPool;
+
+/**
+ * Generator is a program that generates synthetic workloads that try to mimic
+ * real applications that consume the CorfuDb client. Application's access
+ * patterns can be approximated by setting different configurations like
+ * data distributions, concurrency level, operation types, time skews etc.
+ *
+ *
+ * Created by maithem on 7/14/17.
+ */
+public class Generator {
+    public static void main(String[] args) {
+        String endPoint = args[0];
+        int numStreams = Integer.parseInt(args[1]);
+        int numKeys = Integer.parseInt(args[2]);
+        long cpPeriod = Long.parseLong(args[3]);
+        int numThreads = Integer.parseInt(args[4]);
+        int numOperations = Integer.parseInt(args[5]);
+
+        CorfuRuntime rt = new CorfuRuntime(endPoint).connect();
```

Comment:
in the future, we might want to make the number of runtimes an option, and also add capability to reset/reestablish a new runtime